gammapy/data/tests/test_observationlist.py

```diff
@@ -0,0 +1,21 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import absolute_import, division, print_function, unicode_literals
+import numpy as np
+from numpy.testing import assert_allclose
+from astropy.coordinates import SkyCoord
+from ...datasets import gammapy_extra
+from ...utils.testing import requires_data
+from ...utils.energy import EnergyBounds
+from ...data import DataStore, ObservationList
+
+
+@requires_data('gammapy-extra')
+def test_make_psftable():
```

Comment:
Please put this test for the mean psf:
```python
psf1 = PSF from run 1
psf2 = PSF from run 2
psf_mean = mean PSF for both runs

# Add assertions that all three PSFs are similar, e.g.
# - compute the R68 and R95 in the 1 - 10 TeV energy band and see what you get.
# If the values are all similar and sensible, just add asserts on exactly those values (as a regression test).

# Maybe also assert something about the exposure attribute of the mean PSF ... shouldn't it be exactly or approximately the sum of the exposures of the two PSFs?
```
That's enough of a test for the mean PSF for now.